### DISCUSSION TITLE
Difference between this source vs krishnadey30

### DISCUSSION THREAD

**Discussion by aduggal1200** [ssavi-ict/LeetCode-Which-Company#335] [Q&A] [answered]
I want to know if anyone has compared this question source with krishnadey30 GitHub question source. As I am finding that there are some question differences.

https://github.com/krishnadey30/LeetCode-Questions-CompanyWise

How does this source get its questions?

**Comment by ssavi-ict** [comment, marked as answer]
I update the question source manually every day by surfing from Internet. As you can see, the last commit on that repository was around 8 months ago. In my source, you will get updated information. You can see the updation commits here: https://github.com/ssavi-ict/LeetCode-Which-Company/pulls?q=is%3Apr+is%3Aclosed+label%3Acompany-info
